napytau/ingest/model/datapoint_collection.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+from typing import Dict, List, Callable, Iterator
+
+from napytau.ingest.model.datapoint import Datapoint
+from napytau.util.coalesce import coalesce
+from napytau.util.model.value_error_pair import ValueErrorPair
+
+
+class DatapointCollection:
```

Comment:
I know it's an easy to understand data class, but perhaps a short comment would also be useful for the documentation.